HerPublicWebsite.UnitTests/BusinessLogic/Services/QuestionFlowServiceTests.cs

```diff
@@ -100,6 +100,12 @@ public void RunForwardLinkTestCases(QuestionFlowServiceTestCase testCase)
                 QuestionFlowStep.HouseholdIncome, uprn: null
             ),
             QuestionFlowStep.ManualAddress),
+        new(
```

Comment:
With the boiler question move this test is wrong - there are test cases above already though